XamlControlsGallery/ControlExample.xaml

```diff
@@ -46,6 +46,20 @@
                     </VisualState.Setters>
                 </VisualState>
             </VisualStateGroup>
+
+            <VisualStateGroup>
+                <VisualState x:Name="NormalMode"/>
+                <VisualState x:Name="ScreenshotMode">
+                    <VisualState.Setters>
+                        <Setter Target="CodeScrollViewer.Visibility" Value="Collapsed"/>
+                        <Setter Target="ControlPresenter.Padding" Value="{ThemeResource ControlElementScreenshotModePadding}"/>
+                        <Setter Target="ControlPresenter.HorizontalAlignment" Value="Center"/>
+                        <Setter Target="ScreenshotControls.Visibility" Value="Visible"/>
+
+                        <Setter Target="ExampleContainer.Background" Value="Gray"/>
```

Comment:
I'm not a huge fan of hardcoded colors, maybe a theme resource would be better?

Reply:
Agreed, fixed!